views/actions/packages.js

```diff
@@ -115,6 +154,12 @@ function createAddPackageRequest(carrier, trackingNumber) {
   };
 }
 
+function sortHistoryByDate(history) {
+  return history.sort((a, b) => {
+    return dateComparator(a, b);
+  });
```

Comment:
This one is not using db to sort - in case of add package we return the data from the shippo api call, so it would be redundant to wait to store in db and query again